<launch>
        <param name="robot_description" command="cat $(find hsrdp)/umi_rtx_100.urdf" />
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
        <node name="umi_bridge" pkg="hsrdp" type="state_publisher" />
</launch>
